authorization/auth.css

```diff
@@ -0,0 +1,89 @@
+.auth-container {
+  height: 90vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.auth-card {
+  /* width: max-content;
+  max-width: 90%; */
+  width: 30%;
+  min-width: 300px;
+  padding: 2rem;
+  /* border: 1px solid black; */
+  margin: 0 auto;
+
+  box-shadow: rgba(100, 100, 111, 0.2) 0px 7px 29px 0px;
```

Comment:
Just a suggestion, I use box-shadow in many components so for reusability, I make a variable for it. 
example, --box-shadow: rgba(100, 100, 111, 0.2) ;

Reply:
I have been using different box shadows for different components according to situation that is why not adding it to a variable.If same repeats many times ill add that going forward